targets/unity-v2/source/ExampleTestProject/Assets/PlayFabSDK/Client/PlayFabDeviceUtil.cs

```diff
@@ -135,27 +105,6 @@ private static void _OnPlayFabLogin(ClientModels.UserSettings settingsForUser, s
             {
                 settings.DisableFocusTimeCollection = true;
             }
-#endif
-        }
-
-        private static void GetAdvertIdFromUnity(PlayFabApiSettings settings, IPlayFabInstanceApi instanceApi)
```

Comment:
Oh weird... you removed this method, and yet didn't remove the call to it above... Check why this still compiles. It may be #defined out, in which case, you need to enable it to test the removal.